.devcontainer/Dockerfile

```diff
@@ -1 +1,71 @@
-FROM composablefi/composable-devcontainer:latest
+ARG USER=vscode
```

Comment:
Is this Dockerfile working in the end?

Reply:
remote - docker works, baked packages work. nix - does not work. also it presented. does not build of security.
local - docker works, nix works, junod serice node from nix works, etc. builds too long. baking.